src/compute-client/src/logging.rs

```diff
@@ -433,6 +436,11 @@ impl LogVariant {
             LogVariant::Compute(ComputeLog::ShutdownDuration) => RelationDesc::empty()
                 .with_column("worker_id", ScalarType::UInt64.nullable(false))
                 .with_column("duration_ns", ScalarType::UInt64.nullable(false)),
+
+            LogVariant::Compute(ComputeLog::ErrorCount) => RelationDesc::empty()
+                .with_column("export_id", ScalarType::String.nullable(false))
+                .with_column("worker_id", ScalarType::UInt64.nullable(false))
+                .with_column("count", ScalarType::Int64.nullable(false)),
```

Comment:
Add the corresponding foreign keys to `LogVariant::foreign_key`.

Reply:
A bunch of other foreign keys are also missing and it is not clear how declaring them is useful. We should either:

* Do a pass to make sure all foreign keys are correctly and completely defined or
* Stop declaring foreign keys if it's not useful.

In any case I think we should do that separately from this PR. [I've started a discussion in Slack.](https://materializeinc.slack.com/archives/CM7ATT65S/p1696589788239279)